src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color, Math.PI * Math.pow(radius, 2));
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: circle, area: %f, radius: %f, color: %s\n",
+                area,
+                radius,
+                color
+        );
```

Comment:
Use name() for getting String representation of enum constants. Instead of color, which uses the toString() method by default, use color.name() to ensure you get the correct String representation of the enum constant.